jujugui/static/gui/src/app/app.js

```diff
@@ -908,9 +908,9 @@ YUI.add('juju-gui', function(Y) {
           controllerAPI, bakery)}
       />);
 
-      let logoutUrl = '/logout';
+      let logoutUrl = 'logout';
       if (window.juju_config.baseUrl) {
-        logoutUrl = window.juju_config.baseUrl + logoutUrl;
+        logoutUrl = logoutUrl + window.juju_config.baseURL;
```

Comment:
Just noticed this is backwards; will push a fix momentarily. Not sure how that happened. :P